README.md

```diff
@@ -76,6 +76,7 @@ you can activate it. The example above would become:
     require 'active_record'
 
     Datadog.configure do |c|
+      c.tracer enabled: false # optional flag to disable tracer 
```

Comment:
Since this is a quickstart and the default "expected behavior" is to have the tracer enabled (or at least we hope!), can we move this to a different section?

Before "To know if a given framework or lib is supported by our client, please consult our integrations list." we may add:

```
To configure the Datadog Tracer, you can define the `configure` block as follows:
```

```ruby
Datadog.configure do |c|
  c.tracer enabled: false, hostname: 'trace-agent.local'
  # [...]
end
```
```
For a list of available options, check the [Tracer documentation](http://gems.datadoghq.com/trace/docs/#Configure_the_tracer).
```

What do you think?

Reply:
👍

Reply:
Nice! as soon you update the PR I'm going to merge it! thank you very much for pointing out this important issue about documentation!

Reply:
Iv pushed changes. Looks way better